app/common/services/maps.js

```diff
@@ -53,12 +53,17 @@ function (
     var geojsonLayerOptions = {
         onEachFeature: function (feature, layer) {
             var description = feature.properties.description || '',
-                title = feature.properties.title || feature.properties.id;
+                title = feature.properties.title || feature.properties.id,
+                collections = [];
+            for (var i in feature.properties.collections) {
+                collections.push(feature.properties.collections[i].name);
```

Comment:
You could do this easily with underscore `collections = _.pluck(feature.properties.collections, 'name')`